### ISSUE TITLE
DoVi error

### ISSUE THREAD

**Issue by solongandthxforallthefish** [rcdailey/trash-updater#9]
thx for your hard work, this is an awesome project!!

latest release gives me this error, it was working fine yesterday
```
[WRN] One or more of the custom formats you want are duplicated in the guide. These custom formats WILL BE SKIPPED. Radarr requires custom formats names to be unique. Trash Updater is not able to choose which one you actually wanted. This is a bug in the guide and you should request that it be fixed
[WRN] DoVi is duplicated 2 with the following Trash IDs:
[WRN]  - 5d96ce331b98e077abb8ceb60553aa16
[WRN]  - 923b6abef9b17f937fab56cfcf89e1f1

[WRN] The Custom Formats below do not exist in the guide and will be skipped. Names must match the 'name' field in the actual JSON, not the header in the guide! Either fix the names or remove them from your YAML config to resolve this warning
[WRN] ["DoVi"]
```

**Comment by rcdailey** [comment]
So this is actually by design. Remember, it is finding two custom formats in the guide with the same name. When it finds duplicates, it prints the Trash ID of each one. It won't add it for you since it doesn't know which one to pick.

I am currently working on a feature to specify a trash ID when you add it to your YML. I'll keep this open and resolve it once that is done. Thank you for the report!

**Comment by rcdailey** [comment]
Fixed in v1.5.0. See the configuration reference and examples wiki pages for the `trash_ids` property of the configuration.

**Comment by solongandthxforallthefish** [comment]
all working fine now, 
thx for your great work
